### DISCUSSION TITLE
Copilot refuses to work sometimes

### DISCUSSION THREAD

**Discussion by amks1** [community/community#52617] [Copilot Conversations]
### Select Topic Area

Bug

### Body

I have a paid subscription and am using Copilot with VSCode. Version: `v1.82.15`.
It works normally most of the time, and suddenly for intervals ranging from 15 to 30 minutes, it just refuses to work. The logs show this:
```
[INFO] [ghostText] [2023-04-12T11:09:01.651Z] Cancelled after extractPrompt
[INFO] [default] [2023-04-12T11:09:01.900Z] [fetchCompletions] engine https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex
[INFO] [default] [2023-04-12T11:09:02.548Z] request.response: [https://copilot-proxy.githubusercontent.com/v1/engines/copilot-codex/completions] took 647 ms
[INFO] [streamChoices] [2023-04-12T11:09:02.550Z] request done: headerRequestId: [] model deployment ID: []
```

**Comment by Akash1134** [comment]
Hi @amks1 ,

Have you added Copilot to your VS Code settings? 

You need to search for Dev Containers in VS Code Settings, scroll to Dev Containers: Default Extensions and  add "Github.copilot" to that list.

<img width="714" alt="Screenshot 2023-04-12 at 5 50 40 PM" src="https://user-images.githubusercontent.com/95834130/231455275-c2ba002e-6f72-482f-9f84-59a968ec3a13.png">


Hope that helps.
